modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds_{context}"]
+= Integration between shared resources, Insights Operator, and {product-title} Builds
+
+[role="_abstract"]
+Integration between shared resources, Insights Operator, and {product-title} Builds makes using Red Hat subscriptions (RHEL entitlements) easier in {product-title} Builds.
+
+Previously, in {ocp-product-title} 4.9.x and earlier, you manually imported your credentials and copied them to each project or namespace where you were running builds.
+
+Now, in {ocp-product-title} 4.10 and later, {builds-title} can use Red Hat subscriptions (RHEL entitlements) by referencing shared resources and the simple content access feature provided by Insights Operator:
```

Comment:
* Not sure if "Now" is needed
* Add `{nbsp}`
* Add "the" to "Insights Operator"
```suggestion
In {ocp-product-title} 4.10 and later, {builds-title} can use Red{nbsp}Hat subscriptions (RHEL entitlements) by referencing shared resources and the simple content access feature provided by the Insights Operator:
```